ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonSerdeFactory.java

```diff
@@ -44,20 +45,20 @@
 
 @SuppressWarnings("checkstyle:ClassDataAbstractionCoupling")
 @Immutable
-class KsqlJsonSerdeFactory implements SerdeFactory {
+public class KsqlJsonSerdeFactory implements SerdeFactory {
```

Comment:
Ditto.  I imagine we need to make this class public for things to work?

Reply:
I updated it.